app/javascript/packs/stylesheets/analytics.scss

```diff
@@ -72,6 +72,14 @@
   height: 50px; // the height of each row
 }
 
+.analytics-table-flow {
+  margin-top: 26px;
+}
+
+.analytics-table-flow tbody tr {
```

Comment:
Was there a reason for adding this? it makes the rows smaller which I'm not sure is necessary

Reply:
Also for the sake of alignment. Ongoing discussion to resolve with PO team in Slack.

Reply:
Removed https://github.com/SaraAlert/SaraAlert/pull/834/commits/29694316e2a04f2c79c63780fbeafa6ea9e19dbb